AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -313,6 +325,26 @@ class MeetingViewController: UIViewController {
         meetingModel?.resumeCallKitMeeting()
     }
 
+    @IBAction func prevPageButtonClicked(_ sender: UIButton) {
+        meetingModel?.videoModel.getPreviousRemoteVideoPage(completion: { [weak self] success in
+            if success {
+                self?.meetingModel?.videoModel.videoUpdatedHandler?()
+            } else {
+                self?.logger.error(msg: "Failed to get the previous remote video page")
+            }
+        })
```

Comment:
Do we still need to handle callbacks if button is hidden?

Reply:
Discussed offline. Since this is not an async call, we don't need to pass in completion block here. Also, now that we disable the page switching button accordingly, `getPreviousRemoteVideoPage()` and `getNextRemoteVideoPage()` should always success.